backend/chainlit/data/dynamodb.py

```diff
@@ -71,6 +72,32 @@ def _deserialize_item(self, item: Dict[str, Any]) -> Dict[str, Any]:
             for key, value in item.items()
         }
 
+    def _convert_floats_to_decimal(self, obj):
```

Comment:
Can we include thse conversions in `_serialize_item` and `_deserialize_item` itself. This way we do not have to concern ourself with these conversions anywhere else in the code.